### PR TITLE
Small edits to About

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -45,15 +45,15 @@ <h2 class="nodisplay">About</h2>
 		<img data-src="../img/smeidu-selfie-medium.png" alt="Markus Schmeiduch photo" src="../img/smeidu-selfie-medium.png">
 	</div> 
     <div class="nine columns crazy firstchunk"> 
-		<p class="loud">I am a self-employed UX Designer and Product Strategist currently living somewhere between Vienna, <strong>Linz</strong>, and Berlin.</p>
+		<p class="loud">I am a self-employed UX Designer and Product Strategist</br>currently living somewhere between Vienna, <strong>Linz</strong>, and Berlin.</p>
```
Comment (deutschbitte) [inline about/index.html:48]: This word is currently divided on two lines. I'm not sure if this will help or just spill over into a third line.

Comment (smeidu) [reply about/index.html:48]: let's see. if it goes into a third line --- maybe I can adjust the 2nd sentence to a smaller subline font-style.

File: about/index.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h4>CV</h4>
 		</tr>
 		<tr>
 			<td><h5>2012 – 2013</h5></td>
-			<td>Studied Master-level Interaction Design at <a href="https://ciid.dk/">CIID</a></br><i class="half">Copenhagen</i></td>
+			<td>Studied Master-Level Interaction Design at <a href="https://ciid.dk/">CIID</a></br><i class="half">Copenhagen</i></td>
 		</tr>
 		<tr>
 			<td><h5>2010 – 2013</td>
```
Comment (deutschbitte) [inline about/index.html:82]: In this section, the 2001-2004 has a line break and I don't know why. The HTML looks the same as other date ranges so I'm not sure why it's doing this. Any idea?

Screenshot 2022-08-15 at 17 58 24

Comment (smeidu) [reply about/index.html:82]: good catch!
strange indeed! fixed it by giving that table row 2px more width. thx!

**Pull request merged**